Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class TextTable
```

Comment:
This also can be left static, because you don't add any benefits by making it non-static. Just because the homework is OOP, doesn't mean that you have to make everything non-static.